readthedocs/core/resolver.py

```diff
@@ -55,65 +55,77 @@ class ResolverBase:
     """
 
     def base_resolve_path(
-            self,
-            project_slug,
-            filename,
-            version_slug=None,
-            language=None,
-            single_version=None,
-            subproject_slug=None,
-            subdomain=None,
-            cname=None,
-            urlconf=None,
+        self,
+        project_slug,
+        filename,
+        version_slug=None,
+        language=None,
+        single_version=None,
+        project_relationship=None,
+        subdomain=None,
+        cname=None,
+        urlconf=None,
+        custom_prefix=None,
     ):
-        """Resolve a with nothing smart, just filling in the blanks."""
+        """
+        Build a path using the given fields.
+
+        We first build a format string based on the given fields,
+        then we just call ``string.format()`` with the given values.
```

Comment:
Agreed. I'd like to see a couple more links to the proxito README where we explain this stuff in the code as well, so we can remember that's where it's documented, since it's not on https://dev.readthedocs.io/en/latest/